weave-js/src/components/PagePanelComponents/Home/Browse2/RunsTable.tsx

```diff
@@ -27,11 +27,11 @@ import {CategoryChip} from '../Browse3/pages/common/CategoryChip';
 import {CallLink} from '../Browse3/pages/common/Links';
 import {StatusChip} from '../Browse3/pages/common/StatusChip';
 import {renderCell, useURLSearchParamsDict} from '../Browse3/pages/util';
+import {CallSchema} from '../Browse3/pages/wfReactInterface/interface';
 import {
-  CallSchema,
   opVersionRefOpCategory,
   opVersionRefOpName,
-} from '../Browse3/pages/wfReactInterface/interface';
+} from '../Browse3/pages/wfReactInterface/utilities';
```

Comment:
Many files changed in this PR are the result of breaking up a big file into more logical groups.